eclipse-license-resource-bundle/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2018 Red Hat, Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.eclipse.che.parent</groupId>
```

Comment:
do be able to release this artifact it should has relation to parent.
```
+    <parent>
+        <artifactId>maven-parent-pom</artifactId>
+        <groupId>org.eclipse.che.parent</groupId>
+        <version>6.8.0-SNAPSHOT</version>
+        <relativePath>../parent</relativePath>
+    </parent>
```
same as  codestyle/pom.xml

Reply:
It has a relationship to parent. Parent depends on it to resolve the license template file.

You can't have it also depend on parent or you introduce a cyclic dependency. That's why I removed its parent.

Reply:
yes, and that as in issue. I'm sure you would not be able to release it. That is why I want to keep it in a separate repo.

Reply:
um. but if the license resource bundle has no need to depend on parent, then why force the connection? it builds fine. Only thing it doesn't do is self-check its own license header and pom sort order. But do you really want cyclic dependency just so you can guarantee a pom file is clean and has an up to date copyright date?